### PR TITLE
feat: improve Readme MA-834

### PR DESCRIPTION
**Pull request by nanjiangshu**
Improving the README for Metabolic Atlas local deployment

- [x] added table of contents 
- [x] added instruction to add integrated GEMs
- [x] added instruction for how to modify the `.env` file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,30 +29,84 @@ Docker, along with docker-compose, is used to manage the dependencies of this pr
...
+
+### Start docker containers
+Start docker containers of the project by
+```bash
 start-stack
 ```
-The frontend should be available at: `http://localhost/`. If you encounter any problems try running `restart-stack`, or look at the logs `logs backend` / `logs frontend`.
 
-### All helper commands
+Given successful deployment, the frontend should be accessible at: `http://localhost/`. If you encounter any problems try running `restart-stack`, or look at the logs `logs backend` / `logs frontend`.
```
Comment (e0) [inline README.md:100]: In the updated stack, `backend` should be changed to `api`.

File: README.md [hunk 2]
```diff
@@ -11,30 +29,84 @@ Docker, along with docker-compose, is used to manage the dependencies of this pr
...
+
+### Description of helper commands
 
 * To bootstrap the project: `build-stack`
 * To run the project: `start-stack`
 * To display real-time logs: `logs [container-name: frontend/backend/nginx/db]`
```
Comment (e0) [inline README.md:106]: In the updated stack, `backend` should be changed to `api`.

**Review by e0 — CHANGES_REQUESTED** [review]
@nanjiangshu This was great! It should improve the onboarding process a lot for new developers. Just have a small change request noted in the comments about changing `backend` to `api`. It was also nice for me to learn about `popd`.

**Review by e0 — APPROVED** [review]

**Review by mihai-sysbio — COMMENTED** [review]
I've pushed a commit addressing all the points I had (see the extended message of that commit). The main focus was to streamline the Readme by making it more compact.
If all looks good to everyone, I'd say this is ready to be merged.

**Review by mihai-sysbio — APPROVED** [review]

**Comment by e0** [comment]
😕 Just saw these changes.. can't say I agree with the changes from https://github.com/MetabolicAtlas/MetabolicAtlas/pull/543/commits/08cc1ab9bc78b8fb36ea5536e4bf83ac0b5c44a9 , especially the removal of table of contents.

**Comment by mihai-sysbio** [comment]
@e0 let's take this on Slack.

**Pull request merged**